### PR TITLE
Pass gulpfile to ES6

### PR DESCRIPTION
**Pull request by Swiip**
:)

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
+import eslint from 'gulp-eslint';
+import excludeGitignore from 'gulp-exclude-gitignore';
+import babel from 'gulp-babel';
+import nsp from 'gulp-nsp';
+import mocha from 'gulp-mocha';
+import istanbul from 'gulp-istanbul';
+import isparta from 'isparta';
+
+gulp.task('static', () => {
   return gulp.src('src/**/*.js')
```
Comment (zckrs) [inline gulpfile.babel.js]: Not direct related to this PR but can you add gulpfile into process
`  return gulp.src(['gulpfile.babel.js', 'src/**/*.js'])`

**Comment by zckrs** [comment]
`$ npm test` failed in task pre-test since you change gulpfile into gulpfile.babel :/

**Comment by Swiip** [comment]
Oops, running tests, what an idea ^^

**Comment by zckrs** [comment]
:+1: I fixed problem

**Comment by Swiip** [comment]
Thanks ;)

**Pull request merged**